### PR TITLE
As/dummy reducer test file

### PR DESCRIPTION
**Pull request by adamskrodzki**

### CODE REVIEW

File: features/automation/optimization/autoTakeProfit/tests/autoTakeProfitFormChange.integration.test.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+describe('autoTakeProfitFormChangeReducer', () => {
+  describe('user with AutoTakeProfit trigger', () => {
+    it('should recive trigger-id event', () => {})
+  });
+  describe('AutoTakeProfit Add form initialisation', () => {
+    it('should recive execution-price event with properties set to 0', () => {})
```
Comment (johnnieskywalker) [inline features/automation/optimization/autoTakeProfit/tests/autoTakeProfitFormChange.integration.test.ts:6]: just a remark, `defaultAutoTakeProfitData` , `defaultExecutionCollRatio`

File: features/automation/optimization/autoTakeProfit/tests/autoTakeProfitFormChange.test.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+describe('autoTakeProfitFormChangeReducer', () => {
+  it('should modify toCollateral state property on close-type event', () => {})//TODO: this is in more than one file, should be extracted
+  it('should not change unrelated properties on close-type event', () => {})//TODO: this is in more than one file, should be extracted
```
Comment (johnnieskywalker) [inline features/automation/optimization/autoTakeProfit/tests/autoTakeProfitFormChange.test.ts:3]: yeah the fact that is more than one file makes it more complex to test I'll try with enzyme, or if no we can comeback to it when implementing components testing

Comment (adamskrodzki) [reply features/automation/optimization/autoTakeProfit/tests/autoTakeProfitFormChange.test.ts:3]: I think we should refactor Reducers to some form of inheritance

File: Dockerfile [hunk 3]
```diff
@@ -54,6 +54,6 @@ ENV COMMIT_SHA=$COMMIT_SHA \
 COPY . .
 
 RUN chmod +x ./scripts/wait-for-it.sh \
-    && npm run build
```
Comment (adamskrodzki) [inline Dockerfile:57]: Ignore this changes

**Pull request closed without merging**